### DISCUSSION TITLE
Saving .obj with TexturesVertex

### DISCUSSION THREAD

**Discussion by elcronos** [facebookresearch/pytorch3d#537] [Q&A]
I initialized a **Mesh1** with TexturesVertex as:
- I'm overriting the original texture here
`mesh.textures = TexturesVertex(verts_features=torch.clamp(verts_rgb,0.0,1.0))`

Also,I have a second **Mesh2** corresponding to the same object but using the original TextureVertex.

After optimizing **Mesh1** to match the RGB colors of the **Mesh2** using L2 loss. I would like to save the Mesh1 with the learned texture. How can I do that?
